### PR TITLE
US125735/conditionally render discussion/assignment.

### PR DESCRIPTION
**Pull request by mbosiljevac**

### CODE REVIEW

File: components/consistent-evaluation.js [hunk 1]
```diff
@@ -229,7 +237,8 @@ export class ConsistentEvaluation extends LocalizeConsistentEvaluation(LitElemen
 		}
 		this._loading = false;
 		this._setTitle();
-		if (this._telemetry && this._submissionInfo.submissionList) {
+
+		if (this._activityType === assignmentActivity && this._telemetry && this._submissionInfo.submissionList) {
 			this._telemetry.logLoadEvent('consistentEvalMain', this._submissionInfo.submissionList.length);
```
Comment (jeroach) [inline components/consistent-evaluation.js:242]: Hmmm, I wonder if we want to log on CE load regardless of activity type.

Or perhaps we alter the name of the event to `consistentEvalAssignmentsMain`

@vix633 thoughts?

**Review by jeroach — COMMENTED** [review]
Overall lgtm - just the one question on telemetry that I'd like Viktor to weigh in on

**Comment by jeroach** [comment]
Sorry for the conflicts - I bet that's a result of my massive PR

**Review by JamieTran — APPROVED** [review]

File: components/consistent-evaluation.js (hunk 1 above)
Comment (vix633) [reply components/consistent-evaluation.js]: We want to log telemetry regardless of activity type at this level.

File: components/consistent-evaluation.js [hunk 2]
```diff
@@ -194,8 +203,10 @@ export class ConsistentEvaluation extends LocalizeConsistentEvaluation(LitElemen
 		}
 		this._loading = false;
 		this._setTitle();
-		if (this._telemetry && this._submissionInfo.submissionList) {
-			this._telemetry.logLoadEvent('consistentEvalMain', this._submissionInfo.submissionList.length);
+		if (this._activityType === assignmentActivity && this._telemetry && this._submissionInfo.submissionList) {
```
Comment (vix633) [inline components/consistent-evaluation.js:206]: ✅

File: components/helpers/consistent-eval-telemetry.js [hunk 3]
```diff
@@ -37,14 +37,17 @@ export class ConsistentEvalTelemetry {
...
+	async _logUserEvent(href, action, type, performanceMeasureName, activityType, submissionCount) {
 		if (!href || !action || !type || !performanceMeasureName) { return; }
 		const eventBody = new Events.PerformanceEventBody()
 			.setAction(action)
 			.setObject(href, type)
 			.addUserTiming(performance.getEntriesByName(performanceMeasureName))
 			.addCustom('Referer', document.referrer)
 			.addCustom('UserAgent', navigator.userAgent);
+		if (activityType) {
+			eventBody.addCustom('ActivityType', `${activityType}`);
```
Comment (vix633) [inline components/helpers/consistent-eval-telemetry.js:49]: ✅

**Review by jeroach — APPROVED** [review]

**Review by vix633 — APPROVED** [review]
:shipit:

**Pull request merged**